src/gentropy/sumstat_qc_step.py

```diff
@@ -0,0 +1,36 @@
+"""Step to run a finemapping using."""
+
+from __future__ import annotations
+
+from gentropy.common.session import Session
+from gentropy.dataset.summary_statistics import SummaryStatistics
+from gentropy.method.sumstat_quality_controls import SummaryStatisticsQC
+
+
+class SummaryStatisticsQCStep:
+    """Step to run GWAS QC."""
+
+    def __init__(
+        self,
+        session: Session,
+        gwas_path: str,
+        output_path: str,
+        studyid: str,
+    ) -> None:
+        """Run fine-mapping on a studyLocusId from a collected studyLocus table.
```

Comment:
The first line needs to be changed. 

```suggestion
        """Step to calculate quality control metrics on the provided GWAS study.
```

Reply:
I was in a hurry for this PR, sorry for these left-overs from fine-mapping

Reply:
No problem at all. There are reviews for a reason.